taqueria-vscode-extension/src/lib/gui/helpers/SandboxDataHelpers.ts

```diff
@@ -0,0 +1,44 @@
+import { SandboxAccountConfig } from '@taqueria/protocol/types';
+import { ConfigInfo } from '../ObservableConfig';
+
+export type SandboxContract = {
+	alias: string;
+	config: {
+		address: string;
+	};
+};
+
+export type SandboxAccount = {
+	alias: string;
+	address: string;
+};
+
+export function getSandboxContracts(currentConfig: ConfigInfo, sandboxName: string): SandboxContract[] {
```

Comment:
I initially thought this gets accounts from the sandbox (that is, the Flextesa process running in a docker container). A name that emphasizes we get from `config` could be better?